src/scratch/UCERF3/inversion/InversionFaultSystemSolution.java

```diff
@@ -68,7 +68,7 @@
  */
 public class InversionFaultSystemSolution extends SlipEnabledSolution {
 	
-	private InversionFaultSystemRupSet rupSet;
+	protected InversionFaultSystemRupSet rupSet;
```

Comment:
Not important, but could use getRupSet() which is overridden to return type InversionFaultSystemRupSet

Reply:
This too was unneeded, so reverting.